server/src/main/scala/org/http4s/server/armeria/ArmeriaHttp4sHandler.scala

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2013-2020 http4s.org
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.http4s
+package server
+package armeria
```

Comment:
There's an open debate whether this should be `server.$backend` or `$backend.server`.  Blaze is the former, Ember is the latter. @ChristopherDavenport argues strongly for the latter, and I'm inclined to agree.  You're more likely to share internals between the armeria client and server than you are to share client and server internals with other http4s backends, and the flipping this package lets you put those internals in `private[armeria]`, where you can evolve them without breaking bincompat.